### ISSUE TITLE
 tkinter-tooltip-3.1.0 - "Disappear bug " 

### ISSUE THREAD

**Issue by dimmv2** [gnikit/tkinter-tooltip#101]
![python_HkOIvUoDMa](https://github.com/gnikit/tkinter-tooltip/assets/42413898/3870dbcf-2469-4591-9724-c5bdf64e61a1)

tkinter-tooltip-3.1.0 - "Disappear bug" 

sometimes when you hover over the tooltip(button or label ) , and move the mouse out , the full GUI goes to back (lost focus )
 
demo of bug :  ToolTip(userinfo, msg="Actual User",delay=0.5,fg="black",follow=False, bg=tooltip_bg_color)  # dissparaid bug 

i have found the solution to this : 
root.after(250, lambda: ToolTip(userinfo, msg="Actual User", delay=0.5, fg="black", follow=True, bg=tooltip_bg_color)) # fix for desaparition of main gui

maybe you can add these options directly to  "tkinter-tooltip"

**Comment by gnikit** [comment]
Can you please provide a minimal example that reproduces your issue. I can't reproduce.

**Comment by dimmv2** [comment]
here you can see this  : 
`import tkinter as tk
from tktooltip import ToolTip

# Create the root window with a fixed size
root = tk.Tk()
root.title("Right-Click Menu Example")
root.geometry("400x300")  # Set the size of the root window

# User info label example
userinfo = tk.Label(root, text="User Info")
userinfo.pack(pady=10)

# Define the tooltip background color
tooltip_bg_color = "yellow"

# Add the tooltip to the userinfo label with a slight delay
ToolTip(userinfo, msg="Actual User", delay=0.5, fg="black", follow=True, bg=tooltip_bg_color)

# Run the Tkinter event loop
root.mainloop()

**Comment by gnikit** [comment]
@dimmv2 I cannot reproduce the behaviour you described with the provided MWE. I tried on Linux and MacOS, with Python versions 3.10 to 3.12.
That likely means that either there is something wrong with your `tk` installation/machine or there is a bug in your code.
From what I saw in your uploaded animation, you might be overriding your widget's bindings somewhere else.

If you have a MWE that can reproduce this in an another machine or a containerised (controlled) environment, let me know, but I suspect the problem is coming from elsewhere.
